teos-common/build.rs

```diff
@@ -0,0 +1,24 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::configure()
+        .type_attribute(".", "#[derive(serde::Serialize, serde::Deserialize)]")
+        .type_attribute("AppointmentData.appointment_data", "#[serde(untagged)]")
+        .field_attribute("AppointmentData.appointment_data", "#[serde(flatten)]")
+        .field_attribute("appointment_data", "#[serde(rename = \"appointment\")]")
+        .field_attribute("user_id", "#[serde(with = \"hex::serde\")]")
+        .field_attribute("locator", "#[serde(with = \"hex::serde\")]")
+        .field_attribute("encrypted_blob", "#[serde(with = \"hex::serde\")]")
+        .field_attribute("tx", "#[serde(with = \"hex::serde\")]")
```

Comment:
I think this may be related to this: https://github.com/talaia-labs/rust-teos/pull/65/files#diff-58e53499ee6a55d5c9f24d58402715e90aabacf6671d7625650d542ef06f2adc

Not sure if I was trying to wildcard anything containing `tx` and never properly checked if it worked.